openfisca_core/tracers/full_tracer.py

```diff
@@ -121,8 +121,8 @@ def flat_trace(self) -> tracers.FlatTrace:
     def _get_time_in_sec(self) -> float:
         return time.time_ns() / (10**9)
 
-    def print_computation_log(self, aggregate = False):
-        self.computation_log.print_log(aggregate)
+    def print_computation_log(self, depth_max = None, aggregate = False):
```

Comment:
This is a breaking change, I suggest appending after aggregate.

Reply:
Please do !